api/models.py

```diff
@@ -28,3 +31,49 @@ def __init__(self, username: str, email: str, password: str):
 
     def __repr__(self):
         return f'<User {self.username}>'
+
+
+# Companies Models Class:
+class CompaniesModel(db.Model):
```

Comment:
I would suggest maybe "Clients" as the entity terminology here, since it could include possibly academic or things that aren't companies!